### DISCUSSION TITLE
Main Rivers

### DISCUSSION THREAD

**Discussion by MrMikeRose** [digital-land/data-standards-backlog#70] [Planning consideration]
This is the start of a discussion around the Main River data

**Comment by colmjude** [comment]
## Existing data - OS Open River dataset

The [OS Open river dataset](https://osdatahub.os.uk/downloads/open/OpenRivers) includes main rivers as well as other rivers in the UK.

The dataset is available in ESRI Shapfile or GeoPackage formats.

The dataset includes the layers
* hydro_node (points)
* watercourse_link (LineStrong)

**Reply by colmjude** [reply]
The watercourse_link dataset contains 192,065 features.
Looking at a sample, we can see there are over 300 records with the name River Thames.

A typical record looks like this
| field name | value |
| --- | --- |
| fid | 148825 |
| id | FEABD07F-6931-4CFF-8862-FDBA7DF62FB2 |
| flow_direction | in direction |
| length | 2577 |
| fictitious | 0 |
| form | inlandRiver |
| watercourse_name | River Thames |
| watercourse_name_alternative | NULL |
| start_node | ADEC1E5B-E16C-4FE0-B4CE-D422128A97F7 |
| end_node | A9EF45FC-C6D1-4E17-A263-14A970A5D01E |

**Comment by colmjude** [comment]
## Existing data - Statutory Main Rivers

The Environment Agency publish [Statutory Main Rivers](https://www.data.gov.uk/dataset/4ae8ba46-f9a4-47d0-8d93-0f93eb494540/statutory-main-river-map) under OGL.

We had a look at the available data.

### Folder Contents

The following files were found in the folder:

- Statutory_Main_River_MapLine.cst: Size: 10 bytes
- Statutory_Main_River_MapLine.dbf: Size: 55724556 bytes
- Statutory_Main_River_MapLine.prj: Size: 417 bytes
- Statutory_Main_River_MapLine.shp: Size: 57252808 bytes
- Statutory_Main_River_MapLine.shx: Size: 1471372 bytes

Total Folder Size: 114449163 bytes

### Summary

Geometry Type: LineString, MultiLineString
Number of Records: 183,909
Features/Attributes: status, length_km, shape_leng, geometry
Coordinate Reference System (CRS): EPSG:27700
File Size: 57252808 bytes

The `length_km` field ranges from 0 - 13. Median value is 0.09657024

**Comment by jonnyrc** [comment]
The statutory 'Apply for Planning Permission' form asks whether development is within 20 metres of a main river or watercourse. Similarly one of the criteria for Environment Agency consultation is development being within 20 metres of the top bank of a main river.  If this were to become available as a GIS layer then having a 20 metre buffer from the top bank would be very useful in these instances.

**Comment by trodaway** [comment]
FYI, when I've previously enquired about this dataset with the Environment Agency, so I could use it within Idox UniMap Web as a planning constraint, I was advised they're in the process of creating a polygon layer. This will mean a 20m buffer can be applied from the edges, which I would assume are the top banks, to meet the consultation requirement.

The person leading on this project is Dr Sandrine Thomas, sandrine.thomas@environment-agency.gov.uk. In my last update from them, in October 2024, they advised it was expected for completion / publication in early 2025.

**Reply by trodaway** [reply]
This dataset is now live, with the 20m buffer pre-applied, as the Main River Consultation Area: [https://environment.data.gov.uk/dataset/246c6321-9f56-49ce-8022-d36c29078f49](https://environment.data.gov.uk/dataset/246c6321-9f56-49ce-8022-d36c29078f49)
